furious/async.py

```diff
@@ -338,6 +340,12 @@ def start(self, transactional=False, async=False, rpc=None):
         try:
             ret = add(task, transactional=transactional)
         except taskqueue.TransientError:
+            if not retry_transient:
+                raise
+
+            import time
```

Comment:
I'm not sure why with a stdlib one. I would have no issue with this moving to the top of the file. And I agree in general with inline imports. I understand in libs like this were often things in the same file will not execute in the same context. But meh.